### PR TITLE
open-cluster-management/backlog#7551 ddf disable

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: observability/manage_observe.adoc [hunk 1]
```diff
@@ -124,7 +124,18 @@ spec:
 
 * You can disable specific managed clusters. View the following options:
 ** Add the `observability: disabled` label to the custom resource, `managedclusters.cluster.open-cluster-management.io`.
-** From the {product-title-short} console, access a cluster YAML file from the _Clusters_ page to add the `observability: disabled` label.
+** From the {product-title-short} console _Clusters_ page, add the `observability: disabled` label by completing the following steps:
++
+. In the {product-title-short} console navigation, select *Automate infrastructure* > *Clusters*.
+. Select the name of the cluster for which you want to disable the observability collection.
```
Comment (dockerymick) [inline observability/manage_observe.adoc]: Select the name of the cluster for which you want to disable data collection that is sent to observability.

**Review by dockerymick — APPROVED** [review]
Left one comment, but changes look good. Thanks!

File: observability/manage_observe.adoc (hunk 1 above)
Comment (cadawson) [reply observability/manage_observe.adoc]: Thanks!

**Review by bjoydeep — COMMENTED** [review]
This section looks good. However, since the top bullet here says:
`You can disable specific managed clusters`

Should create another bullet in the parent section: **Disable metrics-collector**  that says:
`You can disable metric collection across all the managed clusters where it is enabled now` ?

**Review by bjoydeep — APPROVED** [review]
lgtm

File: observability/manage_observe.adoc [hunk 2]
```diff
@@ -109,10 +109,17 @@ You can also  access Grafana dashboards from the _Clusters_ page. From the navig
 [#disable-metrics-collector]
 == Disable _metrics-collector_
 
-Disable the `metrics-collector` pod to stop data from being collected and sent to the observability service on the {product-title-short} hub cluster. When you disable the `metrics-collector` deployment is scaled to zero, and all managed clusters with the `vendor:OpenShift` label are disabled. View the following options to disable the `metrics-collector`:
+You can disable the metrics-collector, which stops it from collecting the data and sending the collection data to the observability service.
```
Comment (dockerymick) [inline observability/manage_observe.adoc]: You can disable the metrics-collector, which stops it from collecting the data and sending the collected data to the observability service.

Comment (dockerymick) [reply observability/manage_observe.adoc]: Also add the tick marks for the pod name around metrics-collector on line 112  

`metrics-collector`

File: observability/manage_observe.adoc [hunk 3]
```diff
@@ -109,10 +109,17 @@ You can also  access Grafana dashboards from the _Clusters_ page. From the navig
 [#disable-metrics-collector]
 == Disable _metrics-collector_
 
-Disable the `metrics-collector` pod to stop data from being collected and sent to the observability service on the {product-title-short} hub cluster. When you disable the `metrics-collector` deployment is scaled to zero, and all managed clusters with the `vendor:OpenShift` label are disabled. View the following options to disable the `metrics-collector`:
+You can disable the metrics-collector, which stops it from collecting the data and sending the collection data to the observability service. 
+
+[#disable-metrics-collector-on-all-clusters]
+=== Disable metrics-collector on all clusters
```
Comment (dockerymick) [inline observability/manage_observe.adoc]: For consistency, add underscores that surround metric-collector in the section title: _metrics-collector_

File: observability/manage_observe.adoc [hunk 4]
```diff
@@ -122,9 +129,24 @@ spec:
     enableMetrics: false #indicates the observability addon push metrics to hub server
 ----
 
-* You can disable specific managed clusters. View the following options:
-** Add the `observability: disabled` label to the custom resource, `managedclusters.cluster.open-cluster-management.io`.
-** From the {product-title-short} console, access a cluster YAML file from the _Clusters_ page to add the `observability: disabled` label.
+[#disable-metrics-collector-on-a-single-cluster]
+=== Disable metrics-collector on a single cluster
```
Comment (dockerymick) [inline observability/manage_observe.adoc]: Same here for consistency. Italicize metrics-collector in the header

File: observability/manage_observe.adoc [hunk 5]
```diff
@@ -122,9 +129,24 @@ spec:
...
+
+You can disable the metrics-collector on specific managed clusters by completing one of the following procedures:
```
Comment (dockerymick) [inline observability/manage_observe.adoc]: tick marks needed for the pod name: metrics-collector

**Pull request merged**